src/app/pages/LandingPage/components/ArbitrageOpportunity/styled.ts

```diff
@@ -0,0 +1,7 @@
+import styled from 'styled-components';
+
+export const StyledImage = styled.img`
+  width: 30px;
+  height: 30px;
+  object-fit: scale-down;
+`;
```

Comment:
I think we decided to use tailwindcss or css modules for stylings.
Tailwind classes for this solution: `tw-w-8 tw-h-8 tw-object-scale-down`.
Note, tw-w-8 / tw-h-8 is 2rem (32px) but it's ok because by our style guidelines rule was to use 4px (0.25 rem) multiplier for widths/height (28px or 32px in this case)